src/nix/installables.cc

```diff
@@ -56,11 +56,22 @@ SourceExprCommand::SourceExprCommand()
 
 ref<EvalState> EvalCommand::getEvalState()
 {
-    if (!evalState)
+    if (!evalState) {
         evalState = std::make_shared<EvalState>(searchPath, getStore());
+        evalState->initializeRegistries(registryOverrides);
+    }
     return ref<EvalState>(evalState);
 }
 
+EvalCommand::~EvalCommand()
+{
+    for (auto info : evalState->registries[flake::FLAG_REGISTRY]->entries) {
+        if (!info.second.used)
+            warn("the flag flake override %s to %s is not used",
```

Comment:
This sentence doesn't quite parse. Maybe change to something like `unused flag '--flake-override %s %s'`?